starter-code/src/main.js

```diff
@@ -42,9 +44,56 @@ $(document).ready(function(){
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
+
+  $('.card').on('click', function (event) {
+
+
+    event.stopPropagation();
+
+    var thisCard = $(this).children();
+
+    if ($(thisCard[0]).hasClass("back")) {
+
+      $(thisCard[0]).removeClass("back").addClass("front front-active"); //Creating front-active and back-active for comparing in pairs and not turning all cards if you're wrong.
+      $(thisCard[1]).removeClass("front").addClass("back back-active");
+
+      currentCardsAttribute.push(event.currentTarget.id); // Pushing both attributes into array for comparing
+
+
+      if (currentCardsAttribute.length > 0 && currentCardsAttribute.length % 2 === 0) {
```

Comment:
You wanted to compare two cards from the ```currentCardsAttribute``` array so you could check for the length to be equal to 2